src/sql/workbench/parts/notebook/cellViews/output.component.ts

```diff
@@ -54,6 +54,7 @@ export class OutputComponent extends AngularDisposable implements OnInit, AfterV
 
 	ngOnInit() {
 		this._register(this._themeService.onThemeChange(event => this.updateTheme(event)));
+		this.loadComponent();
```

Comment:
This (and removing load from afterviewinit) was a fix after I discovered this was causing a double-creation of the components.